src/moveit_opw_kinematics_plugin.cpp

```diff
@@ -200,7 +203,7 @@ bool MoveItOPWKinematicsPlugin::comparePoses(Eigen::Isometry3d& Ta, Eigen::Affin
   {
     for (int j = 0; j < Ra.cols(); ++j)
     {
-      if (abs(Ra(i, j) - Rb(i, j)) > TOLERANCE)
+      if (std::abs(Ra(i, j) - Rb(i, j)) > TOLERANCE)
```

Comment:
@simonschmeisser turns out there was a pretty significant error in the forward kinematics.... all errors were cast to integer values.

I pushed the small change directly to the melodic branch, but I'm having second thought now on doing this without a pull request, I hope I'm not breaking anything on your side :s

Reply:
interesting that we still found some errors :) No, I guess that change is fine

Reply:
I knew for a while there was an error somewhere, but I could not find him :) Also, now I see why there was no problem with melodic, a C++11 thing probably.